packages/react-composites/tests/browser/call/app/mocks/MockCallAdapter.ts

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { VideoStreamOptions } from '@internal/react-components';
+import { AudioDeviceInfo, PermissionConstraints, VideoDeviceInfo, Call } from '@azure/communication-calling';
+import { RemoteParticipantState, VideoStreamRendererViewState } from '@internal/calling-stateful-client';
+import { CallAdapterState, CallAdapter } from '../../../../../src';
+import { TestCallingState, TestRemoteParticipant } from '../../TestCallingState';
+
+/**
+ * Mock class that implements CallAdapter interface for UI snapshot tests
```

Comment:
Please add detailed docstring about what the mock can / can't do : You can set a snapshot of adapter state on creation, but none of the handlers are implemented, so there is no interactivity.

It'd be helpful to read at a glance what is possible / not possible.